src/ImpactX.H

```diff
@@ -84,6 +84,9 @@ namespace impactx
         //! Delete level data
         void ClearLevel (int lev) override;
 
+        // Resize the mesh, based on the extent of the bunch of particle
```

Comment:
Make sure Doxygen can pick up the string:
```suggestion
        //! Resize the mesh, based on the extent of the bunch of particle
```